paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -87,6 +87,12 @@ public class CoreOptions implements Serializable {
 
     public static final String COLUMNS = "columns";
 
+    public static final ConfigOption<TableType> TYPE =
+            key("type")
```

Comment:
Change key("type") to key("table-type")?

Reply:
No, table-type has been added for hive, and its value is Paimon.

Another type for table.